data/transaction/apiTransactionResult.go

```diff
@@ -42,6 +42,7 @@ type ApiTransactionResult struct {
 	MiniBlockHash                     string                    `json:"miniblockHash,omitempty"`
 	Receipt                           *ReceiptApi               `json:"receipt,omitempty"`
 	SmartContractResults              []*ApiSmartContractResult `json:"smartContractResults,omitempty"`
+	Logs                              *LogsAPI                  `json:"logs,omitempty"`
```

Comment:
We have `ReceiptApi`, `ApiSmartContractResult` and `LogsAPI`. can we use a single pattern?